src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2335,4 +2335,18 @@ Uninstall the following packages: {0}?</value>
     <value>_Extensions</value>
     <comment>Extensions menu</comment>
   </data>
+  <data name="MessagePackageHasDepsInStdLib" xml:space="preserve">
+    <value>{1} has dependencies that conflict with the following Stadard Library packages : {2}.
+    To install {1}, try to first change the package loading order (see https://primer.dynamobim.org/11_Packages/11-Packages.html for more details), then restart {0} to allow changes to take effect, then try and download {1} again.</value>
```

Comment:
did you actually find documentation in the primer on the re-ordering? If it's not in there it would be a good thing to file - we can update the primer and depending on the final design - add in product docs as well.